js/index.js

```diff
@@ -26,4 +28,54 @@ for (let i=0; i < skills.length; i++) {
     skill.innerText = skills[i];
 
     skillsList.appendChild(skill);
-}
\ No newline at end of file
+}
+
+const name = document.querySelector("#name");
+const email = document.querySelector("#email");
+const message = document.querySelector("#message");
+
+
+const messageForm = document.querySelector('#form')
+
+
+messageForm.addEventListener('submit', (event) => {
+
+    event.preventDefault();
+
+    // const nameInput = event.target.name
+    // const emailInput = event.target.email
+    // const messageInput = event.target.message   I couldnt figure out the way the hw meant for me to do
+
+
+    // console.log(nameInput);
+    // console.log(emailInput);
+    // console.log(messageInput);
+    
+    const messageSection = document.querySelector("#messages");
+    
+    const messageList = document.querySelector("#messageslist");
```

Comment:
I think the assignment was asking you use the `messageSection` variable above and not use the `id` of the element. You can actually easily do this by doing by replacing the `document` with `messageSection` and using a `querySelector` of `ul`.